third_party/fontconfig/include/config.h

```diff
@@ -155,8 +155,14 @@
 /* Define to 1 if you have the `random' function. */
 #define HAVE_RANDOM 1
 
+
+#if defined(IS_COBALT_HERMETIC_BUILD)
+/* TODO: (cobalt b/398295440) Add `random_r' support to Evergreen. */
```

Comment:
Is this a long term or short term task? If we want to do this in the near future, I think we should instead use `BUILDFLAG(ENABLE_COBALT_LINUX_HACKS)` instead